curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/return-early-pattern-for-functions.md

```diff
@@ -22,7 +22,7 @@ function myFun() {
 myFun();
 ```
 
-The above outputs "Hello" to the console, returns "World", but `"byebye"` is never output, because the function exits at the `return` statement.
+The above outputs `Hello` to the console, returns `World`, but `byebye` is never output, because the function exits at the `return` statement.
```

Comment:
```suggestion
The above outputs the string `Hello` to the console, returns the string `World`, but the string `byebye` is never output, because the function exits at the `return` statement.
```